relate/static/css/style.css

```diff
@@ -202,26 +202,48 @@ div.progress:hover  span.stats-percentage{
   padding-right:50px;
 }
 
-.grading-page-student-work {
-  position: absolute;
-  top: 60px;
-  left: 0;
-  width: 50%;
-  bottom: 0;
-  padding-left:50px;
-  padding-right:50px;
-  overflow-y: auto;
-}
-
-.grading-page-grade-entry {
-  position: absolute;
-  top: 60px;
-  right: 0;
-  width: 50%;
-  bottom: 0;
-  overflow-y: auto;
-  padding-left:50px;
-  padding-right:50px;
+@media (min-width: 767px) {
```

Comment:
Where does the "767" come from? What about a viewport that's *exactly* 767px wide?